messages/context.json

```diff
@@ -110,6 +110,35 @@
   "pages.editor.store.button.settings.title": "pages.editor.store.button.settings.title",
   "pages.editor.store.button.template.title": "pages.editor.store.button.template.title",
   "pages.editor.store.button.theme.title": "pages.editor.store.button.theme.title",
+  "pages.editor.store.button.back.title": "Back to editor",
+  "pages.editor.store.settings.title": "General Settings",
   "pages.editor.store.settings.error.title": "Something went wrong",
-  "pages.editor.store.settings.error": "Couldn't load store settings."
+  "pages.editor.store.settings.error": "Couldn't load store settings.",
+  "pages.editor.store.settings.refetch": "Try again",
+  "pages.editor.store.settings.pwa.title": "PWA Settings",
+  "pages.editor.store.settings.pwa.error": "Couldn't load PWA settings.",
+  "pages.editor.store.settings.pwa.theme_color": "Theme color",
+  "pages.editor.store.settings.pwa.background_color": "Background color",
+  "pages.editor.store.settings.pwa.start_url": "Start URL",
+  "pages.editor.store.settings.pwa.screen-orientation": "Screen Orientation",
+  "pages.editor.store.settings.pwa.display": "Display",
+  "pages.editor.store.settings.pwa.splash-screen": "Splash screen (iOS)",
+  "pages.editor.store.settings.pwa.android-logo-icon": "Android Logo Icon (512x512)",
+  "pages.editor.store.settings.pwa.ios-logo-icon": "iOS Logo Icon (512x512)",
+  "pages.editor.store.settings.pwa.display.fullscreen": "Fullscreen",
+  "pages.editor.store.settings.pwa.display.standalone": "Standalone (Add to Home Screen)",
+  "pages.editor.store.settings.pwa.display.minimal-ui": "Minimal-UI (Not supported by Chrome)",
+  "pages.editor.store.settings.pwa.display.browser": "Browser",
+  "pages.editor.store.settings.pwa.orientation.any": "Any",
+  "pages.editor.store.settings.pwa.orientation.natural": "Natural",
+  "pages.editor.store.settings.pwa.orientation.landscape": "Landscape",
+  "pages.editor.store.settings.pwa.orientation.landscape-primary": "Landscape (Primary)",
+  "pages.editor.store.settings.pwa.orientation.landscape-secondary": "Landscape (Secondary)",
+  "pages.editor.store.settings.pwa.orientation.portrait": "Portrait",
+  "pages.editor.store.settings.pwa.orientation.portrait-primary": "Portrait (Primary)",
+  "pages.editor.store.settings.pwa.orientation.portrait-secondary": "Portrait (Secondary)",
+  "pages.editor.store.settings.pwa.update-manifest.success": "Manifest successfuly updated!",
+  "pages.editor.store.settings.pwa.update-manifest.error": "Something wrong occurred while updating the manifest...",
+  "pages.editor.store.settings.pwa.update-manifest-icon.success": "App icon successfuly updated!",
+  "pages.editor.store.settings.pwa.update-manifest-icon.error": "Something wrong occurred while updating the App icon..."
```

Comment:
⛏ in this case we use the character `…` instead.

https://www.w3schools.com/charsets/tryit.asp?deci=8230&ent=hellip